doc/conf.py

```diff
@@ -227,19 +232,27 @@
         # Used in quickstart/osm_quickstart
         'https://www.openstreetmap.org/edit',
 
-        # link exists but link check failes
+        # link exists but link check fails
         # SSL: CERTIFICATE_VERIFY_FAILED
         # Used in sponsors.rst
         'https://www.ntua.gr/en',
         'https://www.ice.ucdavis.edu/',
 
-        # Link with certiciate problems
+        # link exists, with strange redirects, but checks fail
+        'https://webchat.freenode.net/',
+
+        # Link with certificate problems
         # Used in quickstart/liblas_quickstart
         #'https://epsg.org/home.html',
         'https://www.epsg.org/',
```

Comment:
```suggestion
        'https://epsg.org/',
```

(the www variant is broken)

Reply:
Updated